packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{! template-lint-disable no-invalid-role }}
+
+<div class={{this.classNames}} {{did-insert this.didInsert}} ...attributes>
+  <div class="hds-tabs__tablist-wrapper">
+    <ul class="hds-tabs__tablist" role="tablist">
+      {{yield
+        (hash
+          Tab=(component
+            "hds/tabs/tab"
+            didInsertNode=this.didInsertTab
+            onClick=this.handleClick
+            onKeyUp=this.handleKeyUp
+            tabIds=this.tabIds
+            panelIds=this.panelIds
+            selectedTabIndex=this.selectedTabIndex
+          )
+        )
+      }}
+      <li class="hds-tabs__tab-indicator" role="presentation"></li>
```

Comment:
I used an li element for the tab-indicator bar. For this reason, I needed to disable the linter using: `{{! template-lint-disable no-invalid-role }}`.

The only way I could see around doing this would be to use a bunch of nested divs instead of a ul list to mark up the tabs. (The individual tabs themselves need to be contained in a parent element with the "tablist" role. The styling which changes the tab-indicator color on hover relies on it being a sibling of the tabs otherwise I could probably use a pseudo selector for the tab-indicator instead.)

Let me know if you don't like this or what you think.

Reply:
The only issue I would've had with it was if the number of items announced by screen readers was altered because of it, but as we use `role="presentation"` the indicator should not be part of the accessibility tree, so no problem there.

Reply:
Thanks, merging this branch for now. I can continue to iterate on this if there are other proposals though.